relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -455,8 +463,36 @@ fn extract_span_metrics(
             }
         }
 
-        // TODO(iker): extract span-specific metrics
-        // still missing: span.operation, description, span.group, module
+        for annotated_span in spans {
+            if let Some(span) = annotated_span.value() {
+                let mut span_tags = shared_tags.clone();
+                // TODO(iker): missing tag: span.group
+
+                if let Some(span_op) = span.op.value() {
+                    span_tags.insert("span.op".to_owned(), span_op.to_owned());
+
+                    let span_module = if span_op.starts_with("http") {
+                        Some("http")
+                    } else if span_op.starts_with("db") {
+                        Some("db")
+                    } else if span_op.starts_with("cache") {
+                        Some("cache")
+                    } else {
+                        None
+                    };
+                    if let Some(module) = span_module {
+                        span_tags.insert("span.module".to_owned(), module.to_owned());
+                    }
+                }
+
+                if let Some(description) = span.description.value() {
+                    // TODO(iker): needs sanitization
+                    span_tags.insert("span.description".to_owned(), description.to_owned());
```

Comment:
Where are these `span_tags` used?

Reply:
The user metric above is logged once per transaction. This is incorrect as per the current requirements, and metrics will be logged per span. This is WIP coming in a follow-up PR.